src/app/ui/TaskBoard.helpers.tsx

```diff
@@ -0,0 +1,7 @@
+import { setActiveWorkflowStateId, setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+
+export const handleAddBtnClicked = (workflowStateId: string) => {
+  store.dispatch(setActiveWorkflowStateId(workflowStateId))
+  store.dispatch(setShowModal())
+}
```

Comment:
I think it is not a good idea to place the helpers for UI state inside the `ui` directory. This might also be useful for the `New Task` button on the top layout. 
We are also not using this directly from the `TaskBoard` component but from the `TaskColumn` and `TaskRow`. 
It is a good idea to abstract away such repetitive UI flows but I think we should make a clear structural decision on where to place such things as there are many more we can abstract away.

Reply:
I've already put this in a separate file in OUT-794 which was created from this branch. Is this better?
![image](https://github.com/user-attachments/assets/eba44117-69eb-49b7-be36-311a3f885566)

Reply:
I mean the same thing where the file reside. Can we revamp?

Reply:
As in.. in the task row and column component itself?
Since this functionality was shared I wanted to not duplicate it across files @aatbip

Reply:
I appreciate the effort to abstract the UI flow, but I think we should reconsider where we're placing this code. The `ui` directory should ideally focus on UI components and their styling. Since the `handleAddBtnClicked` function is more about managing state and could be reused across different components (like the New Task button in the top layout), it would make more sense to place it in a more centralized location, perhaps under a redux or a different directory.

By keeping these helpers outside of the ui directory, we not only maintain a clear separation of concerns but also make the code more reusable and organized. This way, if we need similar functionality elsewhere, we're not tied to a specific UI component structure.
@rrojan